plugins/extsearch/engines/YggTorrent.php

```diff
@@ -131,8 +131,11 @@ public function action($what, $cat, &$ret, $limit, $useGlobalCats)
             );
 
             if ($res) {
+                // Get current URL
+                preg_match('`.+?(?=/torrent)`', $matches["desc"][0], $url);
```

Comment:
We always have absolute URL in  $matches["desc"][0]? Are you sure?

Reply:
Looks like it is. ;)